spring-server-generatedUpdated/src/main/java/io/swagger/api/BookApi.java

```diff
@@ -8,81 +8,103 @@
+
     @ApiOperation(value = "Add a new book to the library", nickname = "addBook", notes = "", authorizations = {
-        @Authorization(value = "petstore_auth", scopes = {
-            @AuthorizationScope(scope = "write:pets", description = "modify pets in your account"),
-            @AuthorizationScope(scope = "read:pets", description = "read your books")
+            @Authorization(value = "petstore_auth", scopes = {
+                    @AuthorizationScope(scope = "write:pets", description = "modify pets in your account"),
+                    @AuthorizationScope(scope = "read:pets", description = "read your books")
             })
-    }, tags={ "book", })
-    @ApiResponses(value = { 
-        @ApiResponse(code = 405, message = "Invalid input") })
-    @RequestMapping(value = "/book",
-        produces = { "application/json" }, 
-        consumes = { "application/json" },
-        method = RequestMethod.POST)
-    ResponseEntity<Void> addBook(@ApiParam(value = "Book object that needs to be added to the store" ,required=true )  @Valid @RequestBody Book body);
+    }, tags = {"book",})
+    @ApiResponses(value = {
+            @ApiResponse(code = 405, message = "Invalid input")})
+    @RequestMapping(value = "/addBook",
+            produces = {"application/json"},
+            consumes = {"application/json"},
+            method = RequestMethod.POST)
+    ResponseEntity<Void> addBook(@ApiParam(value = "Book object that needs to be added to the store", required = true) @Valid @RequestBody Book body);
 
 
     @ApiOperation(value = "Deletes a book", nickname = "deleteBook", notes = "", authorizations = {
-        @Authorization(value = "petstore_auth", scopes = {
-            @AuthorizationScope(scope = "write:pets", description = "modify pets in your account"),
-            @AuthorizationScope(scope = "read:pets", description = "read your books")
+            @Authorization(value = "petstore_auth", scopes = {
+                    @AuthorizationScope(scope = "write:pets", description = "modify pets in your account"),
+                    @AuthorizationScope(scope = "read:pets", description = "read your books")
             })
-    }, tags={ "book", })
-    @ApiResponses(value = { 
-        @ApiResponse(code = 400, message = "Invalid ID supplied"),
-        @ApiResponse(code = 404, message = "Book not found") })
-    @RequestMapping(value = "/book/{bookId}",
-        produces = { "application/json", "application/xml" }, 
-        method = RequestMethod.DELETE)
-    ResponseEntity<Void> deleteBook(@ApiParam(value = "Book id to delete",required=true) @PathVariable("bookId") Long bookId,@ApiParam(value = "" ) @RequestHeader(value="api_key", required=false) String apiKey);
+    }, tags = {"book",})
+    @ApiResponses(value = {
+            @ApiResponse(code = 400, message = "Invalid ID supplied"),
+            @ApiResponse(code = 404, message = "Book not found")})
+    @RequestMapping(value = "/deleteBook/{bookId}",
```

Comment:
this might be better as /book/{bookId}

as the method is Delete no need to have it in the URL